### DISCUSSION TITLE
Best practice about multiple proxies

### DISCUSSION THREAD

**Discussion by etiennejournet** [Kong/kong#7218] [Help] [answered]
Hi,

I'm wondering what are the best practices for multiple proxies or multiple zones on Kubernetes.

I want to expose an external zone for my business API and an internal zone locally accessible for, say, monitoring.
These two zones have separate IP (typically a public one and a private one).
What is the best way to do it ? Two separates kong instances ? Two proxies (is that even possible) ? Something else ?

Thanks for your inputs,

**Comment by mayocream** [comment, marked as answer]
You can have multiple Ingress Controllers for your K8s cluster.
For the public zone, an external Load Balancer (e.g. AWS Network Load Balancer) can be useful, and for the private zone, a NodePort service can be used.
Please reference [multiple-ingress-controllers](https://docs.konghq.com/kubernetes-ingress-controller/2.0.x/concepts/deployment/#multiple-ingress-controllers) for details.
